README.md

```diff
@@ -45,6 +45,14 @@ cops together with the standard cops.
 rubocop --require rubocop-rspec
 ```
 
+Alternatively, use the following array notation when specifying multiple extensions.
+
+```bash
```

Comment:
This is YAML, so in order to make it consistent with the other guides, you may consider writing this as:
```
```yaml
```

Reply:
Appreciate if you could add `yaml` on the line 35.

Reply:
ok :)

Reply:
fixed it!

https://github.com/rubocop-hq/rubocop-rspec/pull/783/commits/1eede967354d99c0ffaf1ca57d7fbfc0c9a1d8cc